client/packages/glsp-sprotty/src/base/command-stack.ts

```diff
@@ -20,12 +20,24 @@ export interface CommandStackObserver {
     beforeServerUpdate(model: SModelRoot): void
 }
 
+export interface IModelUpdateNotifier {
+    registerObserver(observer: CommandStackObserver): boolean | void;
```

Comment:
I guess we should rename CommandStackObserver to ModelUpdateObserver.

Reply:
or IModelUpdateObserver if we choose to go with the I-prefix for interfaces

Reply:
Good idea! I've addressed that the changes below.